### DISCUSSION TITLE
onClick event precedence

### DISCUSSION THREAD

**Discussion by DanSizov** [prabhuignoto/react-chrono#454] [Q&A] [answered]
hi! I'm working currently with your package, thank you very much for your work! I have a question related to timeline. I use `Chrono` as a parent component for the component with button. when I click on the button I only see timeline point is rendered in yellow. no action related to the child component `onClick` event. Can I make a child component taking precedence over the Chrono parent component? I tried `e.stopPropagation()` with different implementations, nothing works.
I checked the discussions for the same issues and didn't find any

**Comment by prabhuignoto** [comment, marked as answer]
@DanSizov This should be fixed in the latest release. can you go ahead and check. let me know if you still face any issues.
